code/modules/mob/living/living_say.dm

```diff
@@ -312,7 +316,7 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 
 	if(speaker != src)
 		if(!radio_freq) //These checks have to be separate, else people talking on the radio will make "You can't hear yourself!" appear when hearing people over the radio while deaf.
-			deaf_message = "[span_name("[speaker]")] [speaker.verb_say] something but you cannot hear [speaker.p_them()]."
+			deaf_message = "[span_name("[speaker]")] [speaker.get_default_say_verb()] something but you cannot hear [speaker.p_them()]."
```

Comment:
To note about this line, previously using `[speaker.verb_say]` here would make it use either the default `verb_say` of the speaker, if it's a human gurgling if it doesn't have a tongue or whatever the tongue `say_mod` is if it does have one, or if it's a basic/simple animal whichever random verb from the list `verb_say` is currently updated to.

Using `[speaker.get_default_say_verb()]` is the closest to previous functionality, and though it means the verb it picks for basic/simple animals will be a different from the original message it doesn't matter much because *you're deaf* and can't hear the original message nor any derivative comms messages anyway.

However, I want to pitch two options here:
If we were to keep `[speaker.verb_say]` now, it now would pretty much always be the default `verb_say` of the speaker, regardless of tongue `say_mod` or lack of tongue or verb randomization, which might be desirable.
We could also replace it by `[message_mods[SAY_MOD_VERB] || speaker.say_mod(message, message_mods)]`, which should maintain previous functionality in addition to allowing deaf `living` instances to see the difference between someone speaking/yelling/singing/whispering/such at them.